database/migrations/2023_01_18_201909_create_voting_tables.php

```diff
@@ -33,13 +29,13 @@ public function up()
         });
         Schema::create('question_options', function (Blueprint $table) {
             $table->id();
-            $table->foreignIdFor(Question::class)->onDelete('cascade');
+            $table->foreignId('question_id')->onDelete('cascade');
```

Comment:
Do not forget to add the foreign constraint here (see [here](https://laravel.com/docs/11.x/migrations#foreign-key-constraints)), IE. `$table->foreignId('question_id')->constrained()`. Same thing applies to all other cases as well.

Edit: I see now that you have just changed `foreignIdFor` to `foreignId` in every migration file, so where the constraint wasn't enforced before it won't be enforced now. So I don't know if adding it is a good idea then, but it may be worth to consider.

Reply:
Yes, I was also wondering this but decided to leave it without it being `->constrained()`. I would rather add it in a new seperate migration.

Reply:
Added to #671